apps/discovery_api/runtime.exs

```diff
@@ -46,14 +49,22 @@ config :discovery_api, DiscoveryApi.Repo,
   port: System.get_env("POSTGRES_PORT"),
   ssl: true,
   ssl_opts: [
-    verify: :verify_peer,
     versions: [:"tlsv1.2"],
-    cacertfile: System.get_env("CA_CERTFILE_PATH"),
+    cacertfile: System.get_env("CA_CERTFILE_PATH")
+  ]
+
+postgres_verify_sni = Regex.match?(~r/^true$/i, System.get_env("POSTGRES_VERIFY_SNI"))
+
+if postgres_verify_sni do
+  config :discovery_api, DiscoveryApi.Repo,
+  ssl_opts: [
```

Comment:
This is like a `concat` or some such? It won't overwrite the `ssl_opts` from above, but add the key/value pairs to the object?